doc/source/whatsnew/v0.25.0.rst

```diff
@@ -172,7 +172,16 @@ Conversion
 Strings
 ^^^^^^^
 
--
+**The ``.str``-accessor performs stricter type checks**
+
```

Comment:
make a new subsection instead, and show an example of previous and current behavior

Reply:
OK. Currently I can only show that things now raise an error that did not before (e.g. trying `.str` on `bytes`). This will then have to be updated once we have a `.bytes` accessor.